01_Core/week03/submissions/exercise 4/index.html

```diff
@@ -0,0 +1,72 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>CSS Exercise 4</title>
+  <meta name="description" content="ITA CSS Layout Exercise">
+  <meta name="author" content="Mark Kroh">
+
+  <style>
+  	.container	{
+		height: 320px;
+		width: 320px;
+		position: relative;
+	  }
+	
+	.interior  {
+		height: 100px;
+		width: 100px;
+	}
+
+	.div1	{
+		background-color: red;
+		position: absolute;
+	}
+
+	.div2	{
+		background-color: orange;
+		position: absolute;
+		top: 50px;
+		left: 50px;
+		border: 5px solid white;
```

Comment:
This modification is used in a few different places. See if there's a way to attach it to a single shared class or tag so that you don't have to repeat yourself several times.

Reply:
Okay this makes sense, I should've added to .interior